packages/yourloops/pages/main-layout.tsx

```diff
@@ -61,7 +63,7 @@ export function MainLayout(): JSX.Element {
 
   return (
     <React.Fragment>
-      {session &&
+      {isAuthenticated && session &&
```

Comment:
I would argue that `isAuthenticated` is unnecessary as it is checked in the main lobby if you want to access these pages :thinking:

Reply:
You're god damn right 🤠